src/app/tests/TestPendingNotificationMap.cpp

```diff
@@ -80,10 +80,10 @@ void TestAddRemove(nlTestSuite * aSuite, void * aContext)
     pendingMap.RemoveAllEntriesForNode(chip::ScopedNodeId());
     uint8_t expectedEntryIndecies[] = { 1, 2, 3, 4, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 };
     iter                            = pendingMap.begin();
-    for (size_t i = 0; i < sizeof(expectedEntryIndecies); i++)
+    for (unsigned char expectedEntryIndecie : expectedEntryIndecies)
```

Comment:
```suggestion
    for (unsigned char expectedEntryIndex : expectedEntryIndices)
```
Part of the miss-spelling is pre-existing, but the automated conversion just made it even worse.

Reply:
renamed to `ch`